src/common/entities/messages.entity.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Entity,
+  PrimaryKey,
+  Property,
+  ManyToOne,
+  EntityRepositoryType,
+  Rel,
+} from "@mikro-orm/core";
+
+import { MessageRepository } from "@/messages/messages.repository";
+
+import { Classroom } from "./classrooms.entity";
+import { CustomBaseEntity } from "./custom-base.entity";
+import { User } from "./users.entity";
+
+@Entity({ tableName: "messages", repository: () => MessageRepository })
```

Comment:
Let's make this plural


```suggestion
@Entity({ tableName: "messages", repository: () => MessagesRepository })
```